### ISSUE TITLE
Redirect Handler doctrine:migrationgenerate "error"

### ISSUE THREAD

**Issue by cwenzelg** [neos/redirecthandler#16]
After adding the package and migration the DB, we get this autogenerated line in every doctrine:migrationgenerate we do:

`$this->addSql('DROP INDEX sourceuripathhash ON neos_redirecthandler_databasestorage_domain_model_redirect');`

Seems that a notification is missing or a table get created by old migrations files but is not used anymore?

**Comment by Sebobo** [comment]
True, but I wonder whether we should keep the index. Will have to check if it's safe to remove it or just a bug in the definition of the model.
